src/components/Card/Card.tsx

```diff
@@ -0,0 +1,53 @@
+import styles from './Card.module.css';
+import Visa from '../../assets/image/Visa.png';
+import MasterCard from '../../assets/image/Mastercard.png';
+import hideNumber from '../../assets/image/CardNumberBlind.svg';
+import { getCardType } from '../../utils/getCardType';
+import { Date } from '../../types/date';
+
+const Card = ({ cardNumbers, date, ownerName }: { cardNumbers: string[]; date: Date; ownerName: string }) => {
+  const displayCardLogo = () => {
+    const cardStyle = styles.logo;
+    const cardType = getCardType(cardNumbers[0]);
+
+    if (cardType === 'VISA') return <img src={Visa} className={cardStyle} />;
+    if (cardType === 'MASTER') return <img src={MasterCard} className={cardStyle} />;
+  };
```

Comment:
```suggestion
  const CARD_IMAGES = {
    VISA: Visa,
    MASTER: MasterCard
  }
  // ...
  
  <img src={CARD_IMAGES[getCardType(cardNumbers[0])]} className={styles.logo} />
```

로 작성한다면 지금보다 훨씬 더 간단하게 만들 수 있을 것 같아요!
조금 복잡하게 느껴진다면 `const cardLogoSrc = CARD_IMAGES[getCardType(cardNumbers[0])` 같이 해놓고 사용해도 될 것 같구요!